### ISSUE TITLE
[10][svk] Browser freezes while long lists of assets load

### ISSUE THREAD

**Issue by wmbutler** [bitshares/bitshares-ui#1643]
**Describe the bug**
Long lists of assets take a while to render

**To Reproduce**
1. Go to a Portfolio with > 100 assets (billbutler - hidden assets tab)
2. Observe the time it takes for scrolling to begin

**Expected behavior**
Screen should show a loading indicator or should load gracefully in the background allowing the user to scroll.

**Comment by gibbsfromncis** [comment]
@svk31 Do you have an idea, does it freeze because of much rerenders or there are some other reason?

**Comment by svk31** [comment]
Partly because of the list itself and partly because of all the market and asset data that needs to be frtched, hard to say which one is more important. You could try disabling the market stats checking when viewing bill's account. 

Pagination/virtualized rendering would solve the issue imo. However, sorting by value for example depends on having all the market data which in turn requires all the rows (corresponding to different assets) to be mounted, so it would not be available in a paginated list unless we also modified the stats checking logic.

**Comment by wmbutler** [comment]
ES could possibly benefit this as well but seems to me that if we stored these assets in localstorage or indexdb and then let the merge of new data into localstorage happen first, we could avoid the browser freeze while fetching.

**Comment by svk31** [comment]
I've been investigating this today, and I've discovered that the main cause of the freezing is the calling of callbacks in the AltStore pubsub model. When you have a ton of assets in your portfolio there can be several hundred callbacks which need to be run every time onGetMarketStats triggers the `emitChange` method of AltStore. Before I added the queuing system that meant for every get_ticker call, now it's every 5 get_ticker calls.

I've been testing different values and it turns out the API node doesn't care if we ask for 5 or 300 get_ticker results, it's very fast no matter what, so I'll be upping the value to 300. I'll include that in a separate commit directly to staging, while develop will get the rest of the improvements as well.
